lighthouse-extension/gulpfile.js

```diff
@@ -178,6 +207,14 @@ gulp.task('watch', ['lint', 'browserify', 'html'], () => {
     'app/src/**/*.js',
     '../lighthouse-core/**/*.js'
   ], ['browserify']);
+
+  gulp.watch([
+    '../lighthouse-core/**/*.html'
```

Comment:
Yes. If we want to change anything, straightway in that precompiled file, this will help. 

Order, yeah we have to run compile first and then browserify.